packages/docs/src/components/page-configs/contribution/ui-element-guide/page-config.ts

```diff
@@ -0,0 +1,18 @@
+import { ApiDocsBlock } from '../../../../types/configTypes'
```

Comment:
May be should use short aliases for imports? As I know in project supports alias `@/`, that match to `src` folder.